### PR TITLE
Signup

### PR DESCRIPTION
**Pull request by Maheshwari-Love**
Sign up button added and link to login button added

### CODE REVIEW

File: Home.html [hunk 1]
```diff
@@ -28,7 +28,7 @@ <h1>
           </ul>
         </div>
         <div class="btns">
-          <button class="login">Login</button>
+          <a  href="./Login/login.html" class="login">Login</a>
```
Comment (Monilprajapati) [inline Home.html:31]: Use Button Tag only Try to add Javascript onClick event

File: Login/login.css [hunk 2]
```diff
@@ -1,5 +1,18 @@
...
 .loginContainer{
-    color: aqua;
+    display: flex;
+    height: 100vh;
+    justify-content: center;
+    align-items: center;
+}
+
+.login{
+    color: rgb(0, 0, 0);
```
Comment (Monilprajapati) [inline Login/login.css]: Change the color to Red

File: SignUp/signup.css [hunk 3]
```diff
@@ -0,0 +1,5 @@
+.signupContainer{
+    color: aqua;
+    font-weight: bold;
+    font-size: large;
+}
```
Comment (Monilprajapati) [inline SignUp/signup.css]: Bring this Text to the center of the Page

**Review by Monilprajapati — CHANGES_REQUESTED** [review]
Follow the Standard coding principles.

File: Login/login.css (hunk 2 above)
Comment (Maheshwari-Love) [reply Login/login.css]: color changed to red

**Review by Monilprajapati — APPROVED** [review]
Practise Practise Practise

File: SignUp/signup.css (hunk 3 above)
Comment (Maheshwari-Love) [reply SignUp/signup.css]: done

**Pull request merged**